pkg/machines/components/vm/vmActions.jsx

```diff
@@ -71,6 +71,20 @@ const VmActions = ({ vm, config, dispatch, onStart, onInstall, onReboot, onForce
         shutdown = DropdownButtons({ buttons: buttons });
     }
 
+    let pause = null;
+    if (config.provider.canPause(state, hasInstallPhase)) {
+        pause = (<button key='action-pause' className="btn btn-default btn-danger" onClick={mouseClick(onPause)} id={`${id}-pause`}>
```

Comment:
You can't use btn-default and btn-danger together. btn-default is what you want. btn-danger would cause the button to be read if it was not overwritten. I see it's used in other places as well. Let's fix it there as well. (not in this commit)

Reply:
I may fix it in another commit, but in this PR, what do you think?

Reply:
It's up to you.

Reply:
Nevermind, just remembered there is another think about VM's states I would like to change, so I will create another PR for those things.

Reply:
Sure, but for this commit at least, don't use the btn-danger. The others fix them when you want.